ui.go

```diff
@@ -43,15 +42,37 @@ func newUI(w fyne.Window, p fyne.Preferences, host func() string) *ui {
 }
 
 func (u *ui) askShutdown() {
-	dialog.ShowConfirm("Shutdown", "Are you sure you want to shut down?",
-		func(ok bool) {
-			if !ok {
-				return
-			}
-
-			cmd := exec.Command("shutdown", "-h", "now")
-			_ = cmd.Start()
-		}, u.win)
+	var (
+		shutdown *widget.Button
+		reboot   *widget.Button
+		cancel   *widget.Button
+		pup      *widget.PopUp
+	)
+	shutdown = widget.NewButton("Power Off", func() {
+		cmd := exec.Command("shutdown", "-h", "now")
+		_ = cmd.Start()
+	})
+	shutdown.Importance = widget.HighImportance
```

Comment:
I'm not going to force you to do this, but I would have personally done it this way in this case:
```suggestion
	shutdown := &widget.Button{Text: "Power Off", Importance: widget.HighImportance, OnTapped: func() {
		cmd := exec.Command("shutdown", "-h", "now")
		_ = cmd.Start()
	}}
```

Reply:
Great, I usually use named parameters for structs but not for functions. I'll keep this in mind for future fyne projects.

Reply:
I'll update the rest to match and keep this in mind going forward

Reply:
It's not anything that we have standardised. I just personally prefer do use the strict initialisation when there are more fields to set than what the functions provide :)